environment.yml

```
@@ -70,7 +70,7 @@ dependencies:
   - xlsxwriter
   - zarr
   # Python packages needed for unit testing
-  - flake8
+  - flake8 ==7.1.0
```

```suggestion
  - flake8 ==5.0.4
```
for compatibility with `prospector`: https://github.com/conda-forge/prospector-feedstock/blob/9273fdcad90787de65c42c3a04dea7e3439909f7/recipe/meta.yaml#L29

---

Done in 130d6f663 😊